streams/piping/general.js

```diff
@@ -154,4 +154,37 @@ promise_test(() => {
 
 }, 'Piping from a ReadableStream for which a chunk becomes asynchronously readable after the pipeTo');
 
+for (const preventAbort of [true, false]) {
+  promise_test(t => {
+
+    const rs = new ReadableStream({
+      pull() {
+        return Promise.reject(undefined);
+      }
+    }, { preventAbort });
+
+    return promise_rejects(t, null, rs.pipeTo(new WritableStream()), 'promise should be rejected');
```

Comment:
I felt it was unnatural to put `null` in there but I concluded that it was the fact that it is rejecting at all that was most important.

I've switched to assert_unreached + assert_equals.